incubator/prometheus-postgresql-adapter/README.md

```diff
@@ -0,0 +1,94 @@
+# prometheus-postgresql-adapter
+
+Installs [prometheus-postgresql-adapter](https://github.com/timescale/prometheus-postgresql-adapter), which provides a remote endpoint to connect a Prometheus service running in Kubernetes to a PostgreSQL endpoint for fast, scalable, long-term metrics storage.
+
+## TL;DR;
+
+```console
+$ helm install incubator/prometheus-postgresql-adapter
+```
+
+## Introduction
+
+This chart bootstraps a [prometheus-postgresql-adapter](https://github.com/timescale/prometheus-postgresql-adapter) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.8+ with Beta APIs enabled
+  - Prometheus instance
+  - PostgreSQL database (recommended to use either [pg_prometheus](https://github.com/timescale/pg_prometheus) or [TimescaleDB](https://github.com/timescale/timescaledb))
+
+## Installing the Chart
+
+You should _at least_ provide the parameter `pgHost`, since prometheus-postgresql-adapter serves as middleware between Prometheus, and a PostgreSQL database. More on that here: [Getting started with Prometheus and TimescaleDB](https://docs.timescale.com/v0.12/tutorials/prometheus-adapter)
+
+To install the chart with the release name `prom-pg-adapter`:
+
+```console
+$ helm install incubator/prometheus-postgresql-adapter --name prom-pg-adapter
+```
+
+The command deploys prometheus-postgresql-adapter on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `prom-pg-adapter` deployment:
+
+```console
+$ helm delete --purge prom-pg-adapter
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the prometheus-postgresql-adapter chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`affinity` | Node affinity for pod assignment | `{}`
+`extraArgs` | Additional server container arguments | `{}`
+`extraEnv` | Additional container environment variables | `{}`
+`image.pullPolicy` | Image pull policy | `IfNotPresent`
+`image.pullSecrets` | Image pull secrets | None
+`image.repository` | Image repository | `timescale/prometheus-postgresql-adapter`
+`image.tag` | Image tag | `0.4`
+`logLevel` | The log level to use [`error`, `warn`, `info`, `debug`] | `info`
+`nodeSelector` | Node labels for pod assignment | `{}`
+`pgDatabase` | The PostgreSQL database | `postgres`
```

Comment:
If possible would be nice to use the same names as command line arguments

Reply:
I was thinking the same thing, but Helm charts generally don't have dots `.` for variables. Referencing the variables inside of the Mustache templates would then require escaping the dot that you all use for command line flags.

See here for more info: https://github.com/helm/helm/blob/master/docs/using_helm.md#the-format-and-limitations-of---set

You'll see why; dots get ugly fast.

Reply:
Basically, I tried as much as possible to match the syntax of the command-line flags with the values in the Helm chart. For example:

pg.host => pgHost
pg.database => pgDatabase